docs/source/dataframe-create.rst

```diff
@@ -76,6 +76,8 @@ storage backend.  You can do this with the ``storage_options=`` keyword:
 
    >>> df = dd.read_csv('s3://bucket-name/my-data-*.csv',
    ...                  storage_options={'anon': True})
+   >>> df = dd.read_parquet('gs://dask-nyc-taxi/yellowtrip.parquet',
+   ...                      storage_options={'anon': True})
```

Comment:
Hmm, I thought the anonymous option for gcs was `{'token': 'anon'}`

Reply:
Thanks for the spot.